scrapers/RealityLovers/RealityLovers.py

```diff
@@ -92,11 +92,40 @@ def sceneByURL():
     log.trace("Scraped the url: " + api_url)
 
     data = scraped.json()
+    # log.debug(json.dumps(data))
 
-    title = data["title"]
+    title = re.sub(r'\s+VR Porn Video$', '', data["title"])
     details = data["description"]
-    image_url = re.sub(r".*,(\S+).*", r"\1", data["mainImages"][0]["imgSrcSet"])
+
+    # image
+    image_urls = [
+        src_set.split(' ')[0] for src_set in data["mainImages"][0]["imgSrcSet"].split(',')
+    ]
+    # pick highest available quality, or fall back to first image
+    image_url = next(
+        (i for i in image_urls if i.endswith('ultra.jpg')),
+        next(
+            (i for i in image_urls if i.endswith('big.jpg')),
+            next(
+                (i for i in image_urls if i.endswith('high.jpg')),
+                next(
+                    (i for i in image_urls if i.endswith('medium.jpg')),
+                    next(
+                        (i for i in image_urls if i.endswith('small.jpg')),
+                        image_urls[0]
+                    )
+                )
+            )
+        )
+    )
```

Comment:
I love a good list comprehension as much as the next man, but I suggest this functionally equivalent solution 😁
```suggestion
    suffixes = ['ultra.jpg', 'big.jpg', 'high.jpg', 'medium.jpg', 'small.jpg']
    image_url = next((url for suffix in suffixes for url in image_urls if url.endswith(suffix)), image_urls[0])
```

Reply:
really nice... I only just discovered the `next` function and didn't realise you could do this "double comprehension", at least you could see what I was trying to do 😁